<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IP/PORT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ip">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.6</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_port">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Socket Message</string>
    </property>
    <widget class="QPushButton" name="chat_btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="chat_txt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="chat_textedit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="chat_btn_open">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Server Monitoring</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RAM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DISK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>461</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 249, 195);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>lcdNumber</zorder>
    <zorder>lcdNumber_2</zorder>
    <zorder>lcdNumber_3</zorder>
    <zorder>lcdNumber_4</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>511</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Download</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>파일다운로드</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>521</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>데이타베이스</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일(&amp;F)</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>편집(&amp;E)</string>
    </property>
    <addaction name="actionConfiguration"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>보기(&amp;V)</string>
    </property>
    <addaction name="actionView_Mode"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>도움말(&amp;H)</string>
    </property>
    <addaction name="actionHelper"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionView_Mode">
   <property name="text">
    <string>View Mode</string>
   </property>
  </action>
  <action name="actionHelper">
   <property name="text">
    <string>Helper</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>chat_txt</tabstop>
  <tabstop>chat_btn</tabstop>
  <tabstop>txt_ip</tabstop>
  <tabstop>txt_port</tabstop>
  <tabstop>btn_connect</tabstop>
  <tabstop>chat_textedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
